module/VuFind/src/VuFind/Controller/AbstractBase.php

```diff
@@ -638,6 +638,18 @@ protected function tagsEnabled()
         return $check->getTagSetting() !== 'disabled';
     }
 
+    /**
+     * Are list tags enabled?
+     *
+     * @return bool
+     */
+    protected function listTagsEnabled()
```

Comment:
Since this is only used in the MyResearchController, does it need to be placed at the AbstractBase level?